enterprise-suite/console-api/static-rules.yml

```diff
@@ -64,3 +64,12 @@
 
 - record: akka_http_http_server_responses_5xx_rate
   expr: irate(akka_http_http_server_responses_5xx[5m]) and akka_http_http_server_responses_5xx offset 1m
+
+- record: kafka_consumer_topic_consumed_rate
+  expr: sum by (app_kubernetes_io_component, app_kubernetes_io_managed_by, app_kubernetes_io_name, app_kubernetes_io_part_of, es_monitor_type, es_workload, namespace, topic) (kafka_consumer_consumer_fetch_manager_metrics_records_consumed_rate)
```

Comment:
Do you mean to have `kafka_consumer_consumer...` (and `producer_producer` below) in there?

Reply:
yep, those are the kafka metrics :-|